indigo_app/views/works.py

```diff
@@ -216,6 +216,7 @@ def get_context_data(self, **kwargs):
     def form_valid(self, form):
         form.instance.updated_by_user = self.request.user
         form.instance.created_by_user = self.request.user
+        form.instance.work_in_progress = True
```

Comment:
I wonder if we should default this to True (after backfilling with False), so that new works are WIP by default? That means we have to explicitly mark them as approved which I think is a bit safer?